.github/workflows/python.yaml

```
@@ -0,0 +1,25 @@
+---
+name: Pylint
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.11", "3.12"]
```

The current Salt versions 3006 and 3007 support Python 3.10.x only, so I'd drop 3.8 and 3.9 here (and add 3.10):

```suggestion
        python-version: ["3.10", "3.11", "3.12"]
```

source: https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-python-version-support.html